docs/source/supported/anvil.rst

```diff
@@ -0,0 +1,111 @@
+================
+Anvil (Purdue)
+================
+
+Platform user guide
+===================
+
+https://www.rcac.purdue.edu/knowledge/anvil
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``purdue.anvil`` - ``SRUN``
+
+* Configuration per node (per platform ID)
+
+  * ``anvil`` queue (1767 nodes)
+
+    * 128 CPU cores
+    * 257 GB of memory
```

Comment:
this should point to our platform ID, so it should be
```
* ``purdue.anvil`` (1767 nodes)
```
and the same for others: `purdue.anvil_mem`, `purdue.anvil_gpu`

also, why it is 1767 nodes? documentation says it is 1000 nodes (https://www.rcac.purdue.edu/knowledge/anvil/architecture)